ui/pages/swaps/loading-swaps-quotes/loading-swaps-quotes.js

```diff
@@ -195,37 +136,9 @@ export default function LoadingSwapsQuotes({
               width="90"
               height="90"
               followMouse={false}
-              lookAtTarget={getMascotTarget(
-                aggregatorNames[quoteCount],
-                midPointTarget,
-                aggregatorLocationMap,
-              )}
+              lookAtTarget={midPointTarget}
             />
           </div>
-          {currentMascotContainer &&
-            midPointTarget &&
-            aggregatorNames.map((aggName) => (
-              <div
-                className={classnames('loading-swaps-quotes__logo', {
-                  'loading-swaps-quotes__logo--transition':
-                    aggName === aggregatorNames[quoteCount],
-                })}
-                style={{
-                  opacity: aggName === aggregatorNames[quoteCount] ? 1 : 0,
-                  top:
-                    aggregatorLocationMap[aggName]?.y + midPointTarget?.y ?? 0,
-                  left:
-                    aggregatorLocationMap[aggName]?.x + midPointTarget?.x ?? 0,
-                }}
-                key={`aggregator-logo-${aggName}`}
-              >
-                <AggregatorLogo
```

Comment:
If we are removing AggregatorLogo from being called we should also remove the files.